app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -109,6 +109,11 @@ def supported_documents
             calculated_fields = tax_calculator.calculate
             receiving_214_credit = calculated_fields[:IT214_LINE_33] > 0
             supported_docs = [
+              {
+                xml: SubmissionBuilder::Ty2022::States::Ny::Documents::RtnHeader,
+                pdf: PdfFiller::Ny201Pdf,
```

Comment:
pdf needs to be nil for the RtnHeader document, if we set `pdf` on two documents it will be included in the combined pdf twice :)

Reply:
right on, fixed. thank you.